### PR TITLE
fix #3914 fix(legacy): Change DC links and buttons to NDT web

### LINKED ISSUE #3914: [Legacy] Change Delivery Console Links and buttons to NDT web
Now that NDT web is here, we should change delivery console links and buttons to redirect to NDT web.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-557)
┆Issue Number: EXP-557

### PR DESCRIPTION
**Pull request by tiftran**
Because

* NDT is the preferred way to view recipes

This commits

* Swaps DC related things to NDT things
* Introduces new environment variables: EXPERIMENTER_INSTANCE and NORMANDY_DEVTOOLS_HOST

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -20,6 +20,7 @@ EMAIL_REVIEW=
 EMAIL_SENDER=
 EMAIL_SHIP=
 EXPERIMENTS_PAGINATE_BY=50
+EXPERIMENTER_INSTANCE=local
```
Comment (jaredlockhart) [inline .env.sample:23]: Can't this just use `HOSTNAME` [https://github.com/mozilla/experimenter/blob/main/.env.sample#L26](here)?

File: .env.sample [hunk 2]
```diff
@@ -30,6 +31,7 @@ KINTO_USER=experimenter
 LOGGING_CONSOLE_LEVEL=INFO
 LOGGING_USE_JSON=False
 NORMANDY_API_HOST=
+NORMANDY_DEVTOOLS_HOST=https://normandy-devtools.services.mozilla.com/
```
Comment (jaredlockhart) [inline .env.sample:34]: Will this be different for different deployments?  Or will it always point to the prod one?

File: .env.sample (hunk 1 above)
Comment (tiftran) [reply .env.sample:23]: oh! is hostname for stage: "stage" and prod: "prod"?

Comment (jaredlockhart) [reply .env.sample:23]: no it's the full url, do you need something that's just 'local' 'stage' 'prod'? how does ndt use that?

File: .env.sample (hunk 2 above)
Comment (tiftran) [reply .env.sample:34]: I don't think it should be different. stage and prod could point to the prod one. One could navigate to stage recipes with the prod devtools instance.

File: .env.sample (hunk 1 above)
Comment (tiftran) [reply .env.sample:23]: yup, devtools urls follow this pattern ext+normandy://<local, dev, stage, prod>/recipes

Comment (jaredlockhart) [reply .env.sample:23]: does ndt use that to figure out which experimenter url to use?

File: .env.sample (hunk 2 above)
Comment (jaredlockhart) [reply .env.sample:34]: If it's always the same then you can just move it right into settings.py and not make it an env var at all?

File: .env.sample (hunk 1 above)
Comment (tiftran) [reply .env.sample:23]: errrm do you mean does experimenter uses it to figure out which ndt url to use?

Comment (jaredlockhart) [reply .env.sample:23]: No like why is it passed to NDT? How does it use it?

File: app/experimenter/settings.py [hunk 3]
```diff
@@ -375,16 +375,19 @@
...
-DELIVERY_CONSOLE_RECIPE_URL = urljoin(DELIVERY_CONSOLE_HOST, "/recipe/{id}/")
-DELIVERY_CONSOLE_EXPERIMENT_IMPORT_URL = urljoin(
-    DELIVERY_CONSOLE_HOST, "/recipe/import/{slug}/"
-)
 NORMANDY_API_HOST = config("NORMANDY_API_HOST")
 NORMANDY_API_RECIPE_URL = urljoin(NORMANDY_API_HOST, "/api/v3/recipe/{id}/")
 NORMANDY_API_RECIPES_LIST_URL = urljoin(NORMANDY_API_HOST, "/api/v3/recipe/")
 
+NORMANDY_DEVTOOLS_HOST = config("NORMANDY_DEVTOOLS_HOST")
+NORMANDY_DEVTOOLS_RECIPE_URL = "{root}{recipe_url}".format(
```
Comment (tiftran) [inline app/experimenter/settings.py:383]: using urljoin makes the "#<instance>" disappear...

Comment (jaredlockhart) [reply app/experimenter/settings.py:383]: Here?  Or you mean the other thing about comments?

**Review by jaredlockhart — COMMENTED** [review]
Looks good 👍 

Did you already file a bugzilla to add the env vars for stage/prod?  Otherwise they won't start after this merges

**Comment by tiftran** [comment]
> Looks good 👍
> 
> Did you already file a bugzilla to add the env vars for stage/prod? Otherwise they won't start after this merges

@jaredlockhart  no, just wanted to ensure this was good to go, before creating a bugzilla ticket, considering that we changed from two env vars to one

**Comment by jaredlockhart** [comment]
@tiftran Yep sounds good, can file bugzilla now 👍

**Comment by tiftran** [comment]
Bugzilla ticket: https://bugzilla.mozilla.org/show_bug.cgi?id=1677921

**Review by jaredlockhart — APPROVED** [review]
@tiftran We can merge this now! 🎉 Thnx @robotblake :D

**Pull request merged**